tools/rtpsrelay/utility.h

```diff
@@ -0,0 +1,64 @@
+#ifndef RTPSRELAY_UTILITY_H_
+#define RTPSRELAY_UTILITY_H_
+
+#include <ace/INET_Addr.h>
+
+#include <sstream>
+
+inline std::string addr_to_string(const ACE_INET_Addr& a_addr)
+{
+  std::array<ACE_TCHAR, 256> as_string{};
+  if (a_addr.addr_to_string(as_string.data(), as_string.size()) != 0) {
+    ACE_ERROR((LM_ERROR, "(%P:%t) %N:%l ERROR: addr_to_string failed to convert address to string"));
+    return "";
+  }
+  return ACE_TEXT_ALWAYS_CHAR(as_string.data());
+}
+
+inline std::string guid_to_string(const OpenDDS::DCPS::GUID_t& a_guid)
```

Comment:
`#include` the required header for the definition of `GUID_t` and declaration of its `operator<<` used 3 lines down